test/unit/util/cuda_matrix_utils.cu

```diff
@@ -13,9 +13,23 @@ namespace micm
           }
       }
 
-      int SquareDriver(CudaVectorMatrixParam& param)
+      void SquareDriver(CudaVectorMatrixParam& param)
       {
           Square<<<param.num_elements_, 1>>>(param.d_data_, param.num_elements_);
       }
+
+      __global__ void AddOne(double* d_data, std::size_t num_elements)
+      {
+          std::size_t tid = blockIdx.x * blockDim.x + threadIdx.x;
+          if (tid < num_elements)
+          {
+              d_data[tid] += 1.0;
+          }
+      }
+
+      void AddOneDriver(CudaVectorMatrixParam& param)
+      {
+        AddOne<<<param.num_elements_, 1>>>(param.d_data_, param.num_elements_);
```

Comment:
Why use `1` rather than the predefined `BLOCK_SIZE` here?

Reply:
Good catch!  Added!